app/posts/register-trainee-teachers/2021-05-08-courses-with-multiple-subjects.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Courses with multiple subjects
+description: Adding support for a second and third subject on a course
+date: 2021-05-08
+---
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+{% from "figure/macro.njk" import appFigure with context %}
+
+When trainees are added to DTTP, up to three subjects can be recorded for their course - these indicate what they were specifically trained to teach in. Register previously only had a single subject, so we’ve added the ability to add a second and third subject.
```

Comment:
This might be clearer: 'When providers add trainees to the DTTP, they can record up to 3 course subjects. These indicate which subjects the trainee was trained to teach in.'